resources/public/api/conf/1.0/application.yaml

```diff
@@ -1,49 +1,135 @@
-openapi: "3.0.3"
-
+openapi: 3.0.3
 info:
-  version: 1.0.0
+  version: 0.13.0
```

Comment:
just a question, where is this version coming from?

Reply:
It looks like it's set from here:
<img width="984" alt="image" src="https://github.com/user-attachments/assets/14c37af9-4e6b-4a10-bf0c-947e3fac977d">
Not entirely sure where this version is/should be set though, might be missing something in the build.sbt? I can have a look to find out exactly what populates this and change it from this arbitrary value

Reply:
Confirmed, adding in 
`version := "1.0.0"`
To the build.sbt changes this value. Will add this value & regenerate the spec now.

Reply:
Awesome, although considering we are not in prod perhaps we should have a 0.1.0 version. @ridow-hmrc, what version do you think?

Reply:
It also brings up a question of how we manage minor and major changes.. do we have to provide a changelog or something?

Reply:
Well it does make an interesting point that it's taking the version of the spec from the version of the _repository_ - so say we have more than 1 API in the repo (Which we will when we have create & amend), do _all_ API version's increment with a changing the version?

Reply:
Discussed this point with Richard, the versioning comes from the git tags versioning and so will get incremented with releases. It just so happened to be at 0.13.0-SNAPSHOT version when I'd created the spec.
As a result, i've removed the explicit `version := "1.0.0` from the build.sbt